package.json

```diff
@@ -16,7 +16,7 @@
     "@heroicons/react": "^2.0.18",
     "@tailwindcss/typography": "^0.5.10",
     "@types/luxon": "^3.3.5",
-    "@types/node": "20.9.2",
+    "@types/node": "20.10.0",
```

Comment:
@coluzziandrea we should also bump the version of this file
https://github.com/latina-in-tech/latina-in-tech.github.io/blob/0873eea19c5d30090d53f208fce7511ef2667d14/.node-version#L1

It's not mandatory but keeping them in sync could avoid possible headache